using_deno_with_other_technologies/node/node_specifiers.md

```diff
@@ -0,0 +1,90 @@
+# Using npm packages with npm specifiers
+
+Deno release 1.25 offers experimental support for npm specifiers. npm specifiers
+allow you to use npm modules directly in Deno with a higher chance of
+compatibility than importing from CDN's, particularly if the modules depend on
+artifact files in their package.
+
+It is important to emphasize that this feature is still under development. **npm
+specifiers are extremely new** and you're likely find scenarios where something
+doesn't work. Please report these problems to the
+[issue tracker](https://github.com/denoland/deno/issues). We'll be working hard
+to improve the compatibility layer and user experience over the next few
+releases.
+
+The way these work is best described with an example:
+
+```ts
```

Comment:
@kitsonk the type checker has been complaining about this example, because `req` and `res` are untyped. Someone else wrote this snippet, I guess we can use the {Request, Response} types from express?

Reply:
Ok. Currently the type checking documentation doesn't appear to support the `npm:` specifiers, which therefore causes problems.  We have to skip/ignore this one for now.

````suggestion
```ts, ignore
````